main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"log"
+	"os"
+	"image"
+	"image/jpeg"
+	"image/color"
+	"yi/point"
+	"yi/harris"
+)
+
+func markPoint(p point.Point, img *image.RGBA64) {
+	red := color.RGBA{
+		R: 1 << 8 - 1,
+		G: 0,
+		B: 0,
+		A: 1 << 8 - 1,
+	}
+	for i := p.X - 10; i <= p.X + 10; i++ {
+		img.Set(i, p.Y, red)
+		img.Set(i, p.Y + 1, red)
+	}
+	for j := p.Y - 10; j <= p.Y + 10; j++ {
+		img.Set(p.X, j, red)
+		img.Set(p.X + 1, j, red)
+	}
+}
+
+func main() {
+	log.SetOutput(os.Stdout)
+	file, err := os.Open("/Users/SU/projects/gillnet/go/src/yi/test-3.jpg")
```

Comment:
This would be way more fun if you accepted a command line parameter instead of hardcoding the file